portfolio/src/main/webapp/comment-form.js

```diff
@@ -1,9 +1,20 @@
-/** Retrieve and post user comments */
+/** Retrieve and post the default quantity of user comments */
 async function getComments() {
-  const response = await fetch('/handle-comment');
-  const comments = await response.json();
+  postFeed(3);
+}
 
-  // Properly format and place the fetched comments within the feed
+/** Update the feed to display the quantity of comments the user requested */
+async function updateComments(button) {
```

Comment:
You don't need to pass `this` into the function, it comes with the `event` as `event.target`

Also this function has no `await`, so `async` keyword is not needed.

```javascript
function updateComments(e) {
  postFeed(e.target.value);
}
```

https://developer.mozilla.org/en-US/docs/Web/API/Event

Reply:
I didn't know event was passed in automatically--that is very convenient

Removed async--I originally thought if the function made a call to an async function, it should also be an async function, but I now see this is only necessary when 'await' is contained within the function.